apollo-client/pom.xml

```diff
@@ -11,7 +11,7 @@
 	<artifactId>apollo-client</artifactId>
 	<name>Apollo Client</name>
 	<properties>
-		<java.version>1.7</java.version>
+		<java.version>1.8</java.version>
```

Comment:
I don't think it's necessary to change the target version from 1.7 to 1.8 simply for this dynamic log level feature

Reply:
ok